scheduler/feasible.go

```diff
@@ -257,6 +368,14 @@ func resolveConstraintTarget(target string, node *structs.Node) (interface{}, bo
 
 // checkConstraint checks if a constraint is satisfied
 func checkConstraint(ctx Context, operand string, lVal, rVal interface{}) bool {
+	// Check for constraints not handled by this iterator.
+	switch operand {
```

Comment:
As we add more constraints that are not handled here I want it to be easily distinguishable which iterator is handling which constraints.